themes/nekolight/nekolight.theme.sh

```diff
@@ -0,0 +1,27 @@
+#! bash oh-my-bash.module
+
+export _omb_prompt_nekolight_version='1.0.0'
+
+symbol=""
```

Comment:
Please prefix `_omb_theme_nekolight_` to the variable name. We want to migrate to the properly namespaced variables. I allowed `icon_*` in the previous nekonight themes just for *backward compatibility* with `cupcake`. When we add new variables, it must be prefixed by the theme namespace.

```suggestion
symbol=""
```